### ISSUE TITLE
Mods sometimes don't load, and don't give friendly error

### ISSUE THREAD

**Issue by misternebula** [amazingalek/owml#399]
Example : misternebula/quantum-space-buddies#330

Preliminary analysis of possible causes :
- `_goHelper` is null
- `_goHelper.CreateAndAdd<IModBehaviour>` returns null

**Comment by misternebula** [comment]
```cs
var mod = _goHelper.CreateAndAdd<IModBehaviour>(modType, helper.Manifest.UniqueName);
_console.WriteLine("Added! Initializing...", MessageType.Debug);
mod.Init(helper);
return mod;
```
NRE happens somewhere here. `helper.Manifest` cannot be null, as it is used earlier without error. Same applies to `_console`

**Comment by misternebula** [comment]
Update : Vesper confirmed that this is a debug-enabled output log :
```
Adding mod behaviour...    OWML    
The script 'ModTemplate.ModTemplate' could not be instantiated!    OWML    
Added! Initializing...    OWML    
Error while adding/initializing Raicuparta.ModTemplate: System.NullReferenceException: Object reference not set to an instance of an object at OWML.ModLoader.Owo.InitializeMod (System.Type modType, OWML.Common.IModHelper helper) [0x0008a] in <14a0e1e29d7b47898d5ec7f3b0893ad4>:0
```

**Comment by misternebula** [comment]
fixed in #395
